test/files/rules/indentwith_tabs.test.ts

```diff
@@ -1,5 +1,8 @@
 // valid code
 module TestModule {
+	/**
+	 * Some documentation
+	 */
```

Comment:
Right -- but I'm not sure it does apply to all multiline comments. Not all multiline comments have asterisks each line. If you don't have asterisks and still want to align with text that you put on the very first line, for instance. I'm not sure that text-inside-multiline-comments should be considered for the same kind of indentation rules.